.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav.vtl

```diff
@@ -112,7 +112,14 @@
     ## Is current page directory $pagePathDir under this $navDirectory? If so,
     ## we found our $navDirectory.
     #if ( $pagePath.contains($dirName) && $pagePathDir == $dirName )
-      #set ( $pageUmbrellaCategory = $umbrellaCategory )
+      ## Hasn't been set yet in a previous iteration
```

Comment:
Can you clarify why this is needed? I figured it by looking at `$umbrellaCategories` but I think it could be done more succinctly with a comment here.

Reply:
Speaking of `$umbrellaCategories` list, I'd recommend moving this line:

    [ 'campus-services/career-development', 'Career', 'default', '/' ]

Under this line:

    [ 'campus-services', 'Campus Services', 'default', '/' ]